examples/vision-camera/modules/webrtc-source/ios/WebrtcSource.podspec

```diff
@@ -0,0 +1,26 @@
+Pod::Spec.new do |s|
+  s.name           = 'WebrtcSource'
+  s.version        = '1.0.0'
+  s.summary        = 'A sample project summary'
+  s.description    = 'A sample project description'
+  s.author         = ''
+  s.homepage       = 'https://docs.expo.dev/modules/'
```

Comment:
Super nit: this example might be copied by someone in future, so maybe we can add some 'fishjam' stuff here? 🤔